src/StreamJsonRpc.Shared/DataContracts/JsonRpcMessage.cs

```diff
@@ -120,6 +127,13 @@ public object[] GetParameters(ParameterInfo[] parameterInfos, JsonSerializer jso
                 return this.Parameters.ToObject<object[]>();
             }
 
+            if (parameterInfos.Length == 1 && parameterInfos[0].ParameterType == typeof(JToken))
+            {
+                // We want to ignore methods that only have one parameter and the parameter type is JToken.
+                // This is reserved to be called when the parameter is passed as an object.
+                return null;
```

Comment:
@ilyabiryukov Is this a breaking change? Do methods today have the option of accepting a parameter as a `JToken`? If not, then I guess this change might be OK, but we may want to revisit in the future since accepting a JToken as a parameter even when the parameters came in as an array sounds like a reasonable scenario to support. So as long as we could support that going forward without a breaking change, I'm OK with this.